honeycombio/honeycomb-opentelemetry-web
# feat: add semantic browser resource fields & honeycomb distro info

## Which problem is this PR solving?
- Closes #10 

## Short description of the changes
Swaps out the `browserDetection` export from `@opentelemetry/resources`, replaces it with the one provided by [`@opentelemetry/opentelemetry-browser-detector`](https://github.com/open-telemetry/opentelemetry-js/blob/57008533aba7ccd51ea80f38ff4f29404d47eb9c/experimental/packages/opentelemetry-browser-detector/src/BrowserDetector.ts) 

## How to verify that this has the expected result
Run the example app. For maximum fields, use a Chromium browser. In honeycomb, queries for the new fields should show your events:
 - `browser.platform`: Chromium only
 - `browser.brands`: Chromium only
 - `browser.mobile`: Chromium only
 - `browser.language`
 - `browser.user_agent`: *note*: this doesn't match the otel semantic conventions of`browser.user_agent.original`
 - `honeycomb.distro.version`: expected value: `0.0.11`
 - `honeycomb.distro.runtime_version`: expected value `browser`
